run.py

```diff
@@ -0,0 +1,7 @@
+import uvicorn
```

Comment:
I usually use a `main.sh` file to do this:
```bash
#!/usr/bin/env bash

export $(egrep -v '^#' .env | xargs)

# run api server
uvicorn strider.server:APP --host 0.0.0.0 --port 5781 --reload
```
I think I just prefer executables to be bash scripts than Python scripts, but are there other pros/cons to doing it either way?

Reply:
The reason for the `run.py` script is so that it can be run with pdb. You can't do `python -m pdb main.sh` but you can do that with `run.py`. This is how FastAPI recommends debugging their applications: https://fastapi.tiangolo.com/tutorial/debugging/